topology/arangodb/collection.go

```diff
@@ -421,6 +430,12 @@ func (c *collection) genericWorker(k string, o DBRecord, done chan *result, toke
 				err = e
 			}
 		}
+	case "down":
+		if _, e := c.topicCollection.RemoveDocument(ctx, k); e != nil {
```

Comment:
Which document you remove? For the peer which has received  peer down notification or for the actual peer that had gone down?

Reply:
Thanks for taking a look. It will remove the document representing the Peer Session for the actual peer that has gone down.

Reply:
Ok, that make sense, thank you.